From f051938d2e80e42f35c1117d1a19ec0da37f3830 Mon Sep 17 00:00:00 2001
From: yzha108 <yanli.l.zhang@intel.com>
Date: Sat, 4 Aug 2018 02:24:06 +0800
Subject: [PATCH 22/54] media: intel-ipu4: ar0231: set max exposure time

Message for Open Source:
Set AR0231 max exposure time to meet 30fps

Message for Internal:
[Issue/Feature]
Set AR0231 max exposure time to meet 30fps

[Root Cause/Changes]
Set valid AR0231 max exposure time to meet 30fps

Change-Id: I5e642b88bacbbc78c669c06b318868c6629fd65a
Tracked-On: #H1407529342
Signed-off-by: yzha108 <yanli.l.zhang@intel.com>
Signed-off-by: Meng Wei <wei.meng@intel.com>
---
 drivers/media/i2c/crlmodule/crl_ar0231at_configuration.h | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/media/i2c/crlmodule/crl_ar0231at_configuration.h b/drivers/media/i2c/crlmodule/crl_ar0231at_configuration.h
index fc639c0..b351c09 100644
--- a/drivers/media/i2c/crlmodule/crl_ar0231at_configuration.h
+++ b/drivers/media/i2c/crlmodule/crl_ar0231at_configuration.h
@@ -1827,7 +1827,7 @@ struct crl_v4l2_ctrl ar0231at_v4l2_ctrls[] = {
 		.name = "T1_COARSE_EXPOSURE_TIME",
 		.type = CRL_V4L2_CTRL_TYPE_INTEGER,
 		.data.std_data.min = 0x0,
-		.data.std_data.max = 0xFFFF,
+		.data.std_data.max = 0x0500,
 		.data.std_data.step = 1,
 		.data.std_data.def = 0x0163,
 		.flags = 0,
@@ -1846,7 +1846,7 @@ struct crl_v4l2_ctrl ar0231at_v4l2_ctrls[] = {
 		.name = "T2_COARSE_EXPOSURE_TIME",
 		.type = CRL_V4L2_CTRL_TYPE_CUSTOM,
 		.data.std_data.min = 0x0,
-		.data.std_data.max = 0xFFFF,
+		.data.std_data.max = 0x0500,
 		.data.std_data.step = 1,
 		.data.std_data.def = 0x0002,
 		.flags = V4L2_CTRL_FLAG_UPDATE,
@@ -1866,7 +1866,7 @@ struct crl_v4l2_ctrl ar0231at_v4l2_ctrls[] = {
 		.name = "T3_COARSE_EXPOSURE_TIME",
 		.type = CRL_V4L2_CTRL_TYPE_CUSTOM,
 		.data.std_data.min = 0x0,
-		.data.std_data.max = 0xFFFF,
+		.data.std_data.max = 0x0500,
 		.data.std_data.step = 1,
 		.data.std_data.def = 0x0001,
 		.flags = V4L2_CTRL_FLAG_UPDATE,
@@ -1886,7 +1886,7 @@ struct crl_v4l2_ctrl ar0231at_v4l2_ctrls[] = {
 		.name = "T4_COARSE_EXPOSURE_TIME",
 		.type = CRL_V4L2_CTRL_TYPE_CUSTOM,
 		.data.std_data.min = 0x0,
-		.data.std_data.max = 0xFFFF,
+		.data.std_data.max = 0x0500,
 		.data.std_data.step = 1,
 		.data.std_data.def = 0x0,
 		.flags = V4L2_CTRL_FLAG_UPDATE,
-- 
1.9.1

